### PR TITLE
Adding back processCpuTime metric

### LINKED ISSUE #9384: LG-248: Add new metric representing Process CPU Time to mpMetrics-2.0
new Gauge metric called Process CPU Time, which is getProcessCpuTime from the OSMXBean https://docs.oracle.com/javase/7/docs/jre/api/management/extension/com/sun/management/OperatingSystemMXBean.html
description is “Returns the CPU time used by the process on which the Java virtual machine is running in nanoseconds.”

need to take care as this is implemented differently on different JVMs

### PR DESCRIPTION
**Pull request by jennifer-c**
fixes #9384 

#build

### CODE REVIEW

File: dev/com.ibm.ws.microprofile.metrics.2.0/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops [hunk 1]
```diff
@@ -99,7 +99,7 @@ cpu.systemLoadAverage.description=Displays the system load average for the last
 cpu.processCpuLoad.description=Displays the 'recent cpu usage' for the Java Virtual Machine process.
 
 #Cpu processCpuTime message
-#cpu.processCpuTime.description=Displays the CPU time used by the process on which the Java Virtual Machine is running.
+cpu.processCpuTime.description=Displays the CPU time used by the process on which the Java Virtual Machine is running.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.microprofile.metrics.2.0/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops]: Change
"Displays the CPU time used by the process on which the Java Virtual Machine is running."
To
"Displays the CPU time that is used by the process on which the Java virtual machine is running."

Comment (jennifer-c) [reply dev/com.ibm.ws.microprofile.metrics.2.0/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops]: Done, thanks!

**Comment by jennifer-c** [comment]
#build 

#run-libby-bot

**Comment by jennifer-c** [comment]
#build

**Pull request merged**